examples/programmatic/llpr_forces/force_llpr.py

```diff
@@ -53,6 +56,9 @@
         "reader": "ase",
         "key": "energy",
         "unit": "kcal/mol",
+        "type": "scalar",
+        "per_atom": False,
+        "num_properties": 1,
```

Comment:
Isn't `num_properties` for a scalar always one?

Reply:
It can be more than one, for example in a DOS, where you're predicting many numbers and they're all scalars (invariants). Or SOAP, which is also a scalar representation made up of many scalars

Reply:
I see, maybe `num_properties` is not the best name then. I know why you chose it. At least explain this somewhere.